app/assets/javascripts/discourse/controllers/topic.js.es6

```diff
@@ -649,11 +649,37 @@ export default Ember.Controller.extend(SelectedPostsCount, BufferedContent, {
       const quotedText = Quote.build(post, quoteState.get('buffer'));
       this.send('deselectText');
 
-      composerController.open({
-        action: Composer.CREATE_TOPIC,
-        draftKey: Composer.REPLY_AS_NEW_TOPIC_KEY,
-        categoryId: this.get('model.category.id')
-      }).then(() => {
+      var options;
+
+      if (this.get('model.isPrivateMessage')) {
+
```

Comment:
I'll prefer if we :scissors: these empty lines. A good rule of thumb is to follow the style of the surrounding code.